torch/_torch_docs.py

```diff
@@ -597,6 +597,42 @@ def parse_kwargs(desc):
             [ 0.,  0.,  0.]])
 """)
 
+add_docstr(torch.bincount,
+           r"""
+bincount(self, weights=None, minlength=0) -> Tensor
+
+Count the frequency of each value in an array of non-negative ints.
+
+The number of bins (size 1) is one larger than the largest value in
+:attr:`input`. If :attr:`minlength` is specified, the number of bins is at least
+:attr:`minlength`. If ``n`` is the value at position ``i``,
+:math:`out[n] += weights[i]` if :attr:`weights` is specified else
+:math:`out[n] += 1`.
+
+Arguments:
+    input (Tensor): 1-d int tensor
+    weights (Tensor): optional, weight for each value in the input tensor.
+        Should be of same size as input tensor.
+    minlength (int): optional, min number of bins. Should be non-negative.
+
+Shape:
+    output (Tensor): ``Size([max(input) + 1])``
+
+Example::
+
+    >>> input = torch.randint(0, 8, (5,), dtype=torch.int64)
+    >>> weights = torch.linspace(0, 1, steps=5)
+    >>> input, weights
+    (tensor([4, 3, 6, 3, 4]),
+     tensor([ 0.0000,  0.2500,  0.5000,  0.7500,  1.0000])
+
+    >>> torch.bincount(input)
+    tensor([0, 0, 0, 2, 2, 0, 1])
+
+    >>> input.bincount(weights)
```

Comment:
should this be `input.bincount(input, weights)` ?

Reply:
No. It is similar to defining a method in a class. You'd define the method:
```
class Abc:
  def cmethod(self, ...):
      pass
```
And you'd call the `cmethod` as: 
```
cls = Abc()
cls.cmethod()  # and not, cls.cmethod(cls) 
```
Here, `bincount` is a method for the `Tensor` class and `input` is a `Tensor` instance.

Reply:
It should be `>>> print(input.bincount(weights))` not `>>> print(`~~`torch`~~`.bincount(weights))`. As such bincount is only implemented for integral tensors as mentioned in the error.

Reply:
ah, I see! sorry I missed that one... All is good now! Thank a lot!